peachjam/migrations/0152_backfill_natures.py

```diff
@@ -22,6 +22,7 @@ def forwards(apps, schema_editor):
         # get_or_create with defaults={name: name} doesn't seem to work
         if not nature.name:
             nature.name = name
+            nature.name_en = name
```

Comment:
@longhotsummer I've added it

Reply:
I've done a test deployment and this works.